main/mkinitfs/APKBUILD

```diff
@@ -9,18 +9,15 @@ arch="all"
 license="GPL-2.0"
 # currently we do not ship any testsuite
 options="!check"
-makedepends_build=""
 makedepends_host="busybox kmod-dev util-linux-dev cryptsetup-dev linux-headers"
-makedepends="$makedepends_build $makedepends_host"
+makedepends="$makedepends_host"
```

Comment:
If we're removing `$makedepends_build`, can't we just throw the contents of `$makedepends_host` into `$makedepends` and call it a day?

Reply:
i think those are used by `scripts/bootstrap.sh`.